pkg/apis/options/legacy_options_test.go

```diff
@@ -21,9 +21,10 @@ var _ = Describe("Legacy Options", func() {
 			legacyOpts.LegacyUpstreams.PassHostHeader = true
 			legacyOpts.LegacyUpstreams.ProxyWebSockets = true
 			legacyOpts.LegacyUpstreams.SSLUpstreamInsecureSkipVerify = true
-			legacyOpts.LegacyUpstreams.Upstreams = []string{"http://foo.bar/baz", "file://var/lib/website#/bar"}
+			legacyOpts.LegacyUpstreams.Upstreams = []string{"http://foo.bar/baz", "file://var/lib/website#/bar", "static://204"}
```

Comment:
Are these intentionalle only two slashes after `file:`?

Reply:
I don't think it particularly matters in this context, but this lead me to find a second related bug, so great spot! I've pushed another commit which I think fixes the conversion of the file based upstreams